.github/renovate.json

```diff
@@ -1,6 +1,11 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"automerge": true,
+	"extends": [
+		"config:recommended",
+		":maintainLockFilesWeekly",
```

Comment:
[Question] This feature looks nice (TIL, thanks!) but it doesn't look to me like it respects `minimumReleaseAge`. That's a security risk. Unless I'm missing something (am I?), I'd say let's drop it for now.

```suggestion
```

Aside: I've been thinking of bumping `minimumReleaseAge` to `7 days`. #1463

Reply:
Oh, and:

> I'm not sure if `maintainLockFilesWeekly` is needed when there is pnpm dedupe in every PR already

It looks to me that `maintainLockFilesWeekly` will update packages, not just dedupe them. So that is a nice extra.